lib/managers/events.js

```diff
@@ -156,4 +166,107 @@ Events.prototype.getEventStream = function(streamPosition, callback) {
 	});
 };
 
+/**
+ * Create a stream of enterprise events.
+ *
+ * By default, the stream starts from the current time.
+ * Pass 'startDate' to start from a specific time.
+ * Pass 'streamPosition' to start from a previous stream position, or '0' for all available past events (~1 year).
+ * Once the stream catches up to the current time, it will begin polling every 'pollingInterval' seconds.
+ * If 'pollingInterval' = 0, then the stream will end when it catches up to the current time (no polling).
+ *
+ * @param {Object} [options] - Options
+ * @param {string} [options.streamPosition] - The stream position to start from (pass '0' for all past events)
+ * @param {string} [options.startDate] - The date to start from
+ * @param {string} [options.endDate] - The date to end at
+ * @param {string} [options.eventTypeFilter] - Comma-separated list of event types to return
+ * @param {int} [options.pollingInterval=60] - Polling interval (in seconds).  Pass 0 for no polling.
+ * @param {int} [options.chunkSize=500] - Number of events to fetch per call (max = 500)
+ * @returns {EnterpriseEventStream} - The enterprise events stream
+ */
+Events.prototype.getEnterpriseEventStream = function(options) {
+	return new EnterpriseEventStream(this.client, options);
```

Comment:
This method is synchronous, where the User Events stream is asynchronous.  Could we make them both follow the same convention?